ts/src/base/Exchange.ts

```diff
@@ -3363,11 +3363,21 @@ export default class Exchange {
         return this.parseNumber (value, defaultNumber);
     }
 
-    parsePrecision (precision: string) {
+    parsePrecision (precision) {
+        /**
+         * @ignore
+         * @method
+         * @param {int} precision The number of digits to the right of the decimal
+         * @returns {string} a string number equal to 1e-precision
+         */
         if (precision === undefined) {
             return undefined;
         }
-        return '1e' + Precise.stringNeg (precision);
+        let parsedPrecision = '0.';
```

Comment:
@samgermain we gotta be careful when we do these kinds of global edits, if I merged it without proper testing, we would break the order placement across the entire lib in Python because `parsePrecision` is provided with a string, and here you assume it's an integer.